### PR TITLE
[EMCAL-851] Add postprocessing at asynchronous level for EMCAL

### PR DESCRIPTION
**Pull request by mfasDa**

### CODE REVIEW

File: DATA/production/qc-postproc-async/emc.json [hunk 1]
```diff
@@ -0,0 +1,398 @@
+{
+    "qc": {
+        "config": {
+            "database": {
+                "implementation": "CCDB",
+                "host": "alio2-cr1-hv-qcdb-gpn.cern.ch:8083",
```
Comment (knopers8) [inline DATA/production/qc-postproc-async/emc.json:6]:
```suggestion
                "host": "ali-qcdb-gpn.cern.ch:8083",
```

File: DATA/production/o2dpg_qc_postproc_workflow.py [hunk 2]
```diff
@@ -78,6 +78,7 @@ def add_QC_postprocessing(taskName, qcConfigPath, needs, runSpecific, periodSpec
 
   ## The list of QC Post-processing workflows, add the new ones below
   add_QC_postprocessing('example', 'json://${O2DPG_ROOT}/DATA/production/qc-postproc-async/example.json', needs=[], runSpecific=False, periodSpecific=False, passSpecific=True)
+  add_QC_postprocessing('EMC', 'json://${O2DPG_ROOT}/DATA/production/qc-postproc-async/emc.json', needs=[], runSpecific=False, periodSpecific=False, passSpecific=True)
```
Comment (knopers8) [inline DATA/production/o2dpg_qc_postproc_workflow.py:81]: FYI, this means that the trends will be done for all the runs and periods which have the specific pass, e.g. `apass2`.

**Review by knopers8 — COMMENTED** [review]
It looks OK, please use the alias for the host name and we should be good to go.

File: DATA/production/qc-postproc-async/emc.json (hunk 1 above)
Comment (mfasDa) [reply DATA/production/qc-postproc-async/emc.json:6]: Leftover from testing, will be fixed.

File: DATA/production/o2dpg_qc_postproc_workflow.py (hunk 2 above)
Comment (mfasDa) [reply DATA/production/o2dpg_qc_postproc_workflow.py:81]: Ideally we need the trending for each period and pass separately. How do we do this?

Comment (mfasDa) [reply DATA/production/o2dpg_qc_postproc_workflow.py:81]: OK, thanks, will modify.

**Review by chiarazampolli — APPROVED** [review]

**Comment by mfasDa** [comment]
Second (squash-merged commit) includes both changes requested by @knopers8

**Pull request merged**